### PR TITLE
Updated UI/UX for auth pages

### PR DESCRIPTION
**Pull request by mohammedali101**
- Added CircularProgressIndicator (loading spinner) on all auth pages
- Fixed spelling and removed unnecessary comments

UPDATED:
- loading spinner works in both instances (refer to videos)

https://user-images.githubusercontent.com/91173297/203869023-7aac0c19-ec4a-4fa3-a822-98ecb223b4e3.mov

https://user-images.githubusercontent.com/91173297/203869030-c24e8d69-6203-4e9b-bcef-5c142c4e5462.mov

### CODE REVIEW

**Comment by abe-mused** [comment]
So when there's an error, it keeps spinning?

File: lib/auth_pages/sign_in.dart [hunk 1]
```diff
@@ -142,7 +151,8 @@ class _LoginPageState extends State<LoginPage> {
                           Navigator.push(
                             context,
                             MaterialPageRoute(
-                              builder: (context) => const SignUpPage(),
+                              builder: (context) => 
+                              const SignUpPage(),
```
Comment (abe-mused) [inline lib/auth_pages/sign_in.dart:155]: keep on same line

File: lib/auth_pages/sign_in.dart [hunk 2]
```diff
@@ -130,8 +130,17 @@ class _LoginPageState extends State<LoginPage> {
                         style: TextButton.styleFrom(
                           padding: const EdgeInsets.symmetric(vertical: 5, horizontal: 60),
                         ),
-                        onPressed: doLogIn,
-                        child: const Text("Sign In")),
+                        onPressed: () async {
+                            setState(() {
+                              _isUpdatingSignIn = true;
+                              doLogIn();
```
Comment (abe-mused) [inline lib/auth_pages/sign_in.dart:136]: take `doLogIn()` out of `setState`

File: lib/auth_pages/sign_up.dart [hunk 3]
```diff
@@ -65,6 +63,11 @@ class _SignUpPageState extends State<SignUpPage> {
 
   @override
   Widget build(BuildContext context) {
+    if (_isUpdatingSignUp) {
+      const Center(
+          child: CircularProgressIndicator(),
+        );
+    }
```
Comment (abe-mused) [inline lib/auth_pages/sign_up.dart:70]: Why do we need this? (we already have the `CircularProgressIndicator` below)

**Review by abe-mused — CHANGES_REQUESTED** [review]

**Comment by mohammedali101** [comment]
> So when there's an error, it keeps spinning?

UPDATED:
 Future.delayed(const Duration(seconds: 1), (){
                              setState(() {
                                _updateResetPassword = false;
                              });

**Review by abe-mused — APPROVED** [review]
lgtm!

**Pull request merged**